src/prepare.ts

```diff
@@ -117,21 +117,15 @@ ${user[filePath]}
       test[filePath] = "finishSuite()";
     }
 
-    const lines = test[filePath].trim().split("\n");
-
     // Delete globals import
-    const globalsImportLineIndex = lines.findIndex(
-      (l) =>
-        l.indexOf("import ") !== -1 && l.indexOf("from '@jest/globals'") !== -1,
-    );
-    if (globalsImportLineIndex !== -1) {
-      lines.splice(
-        globalsImportLineIndex,
-        1,
-        "// import { ... } from '@jest/globals'",
-      );
-    }
-
+    const lines = test[filePath]
+      .trim()
+      .replace(
+        /import.*from\s+['"]@jest\/globals['"];?/s,
```

Comment:
```suggestion
        /import\s+.*?\s+from\s+['"]@jest\/globals['"];?/s,
```

`.*` is greedy and `.*?` is not. This fixes a case where:

```javascript
import ... from 'something';
import ... from '@jest/globals';
```